.project_automation/functional_tests/entrypoint.sh

```diff
@@ -30,11 +30,11 @@ run_test() {
     if [ -z "$1" ]; then
         echo "Running e2e test: ALL"
         taskcat test run -n
-        .project_automation/functional_tests/scoutsuite/scoutsuite.sh
+        # .project_automation/functional_tests/scoutsuite/scoutsuite.sh
     else
         echo "Running e2e test: $1"
         taskcat test run -n -t $1
```

Comment:
@ryanjpayne we need to remove the "-n" since we're not running scoutsuite so it cleans up the cloudformation deployments from the tests before running the next one. change it to taskcat test run -t $1